client/src/containers/CampaignAdmin/CampaignAdminList.js

```diff
@@ -274,13 +279,69 @@ class CampaignAdminList extends SilverStripeComponent {
         </div>
       );
     } else {
-      preview = <Preview itemLinks={itemLinks} itemId={itemId} onBack={this.handleCloseItem} />;
+      preview = (
+        <Preview
+          itemLinks={itemLinks}
+          itemId={itemId}
+          onBack={this.handleCloseItem}
+          moreActions={this.getMoreActions()}
+        />
+      );
     }
 
     return preview;
   }
 
   /**
+   * @return {array}
+   */
+  getMoreActions() {
+    const selectedItem = this.getSelectedItem();
+
+    if (!selectedItem) {
+      return null;
+    }
+
+    const requiredByNum =
+      selectedItem._links.referenced_by &&
+      selectedItem._links.referenced_by.length || 0;
+    const unremoveableInfoText = requiredByNum < 2 ?
+      i18n._t(
+        'CampaignAdmin.UNREMOVEABLE_INFO_SINGULAR',
+        'Required by %s item, and cannot be removed directly.'
+      ) :
+      i18n._t(
+        'CampaignAdmin.UNREMOVEABLE_INFO_PLURAL',
+        'Required by %s items, and cannot be removed directly.'
+      );
```

Comment:
I strongly don't agree with having plurals hardcoded like this... we've talked about this before as well. This doesn't translate well for languages with different pluralisation rules to english (like Russian) and duplication work for no pluralisation rules (like Chinese).

Please change this to `'Required by {number} item(s), and cannot be removed directly.'`

If you'd like we can work on (or raise a story for) a pluralisation API for the javascript side, which can be similar to what was implemented on the PHP side.

Missing entry for new translation strings in `en.js` and `en.json`

Reply:
Sorry, I totally forgot about this. I've created an issue for the API (https://github.com/silverstripe/silverstripe-admin/issues/218).